### DISCUSSION TITLE
We should advertise non-georust crates on georust.org

### DISCUSSION THREAD

**Discussion by michaelkirk** [georust/georust.github.io#14] [Ideas]
Currently, https://georust.org serves two purposes:

1. links to our communication channels 
2. curates a subset of all the [georust github organization's repositories](https://github.com/orgs/georust/repositories). 

I wanted to talk about 2.

In discord earlier in the week, a user expressed hesitation to use a crate because, by not being listed on that site, it seemed less likely to work in the georust ecosystem.

It's true that one of the benefits of moving a crate into the georust organization is that other people can help to maintain it - so a crate is less likely to get abandoned. But it's also true that there are many well maintained useful GIS crates outside of the georust organization.

https://github.com/pka/awesome-georust does a great job of aggregating useful tools for people doing geospatial in rust, including many georust and non-georust crates.

I think our users would be better served by seeing something more like that.

We could simply replace our listing with a link to https://github.com/pka/awesome-georust or embed it's contents into our site.
@pka has done a great job of managing that list, so I'm especially eager to hear what their thoughts/objections would be.

Another future direction for the site could be to include some longer form walkthroughs/tutorials that might involve multiple crates. E.g. parsing some json, doing some analysis in geo, and writing the results. This would need to be maintained, so we'd want to be deliberately minimal in order to have half a chance of keeping things up to date. As some of these are likely to exist in the ecosystem already, maybe curating a list of links to existing tutorials would be easier.

**Comment by pka** [comment]
Advertising spatial rust crates is the primary goal of the list, so I'm supporting any use of it. I'm fine with keeping awesome-gerust in my personal account and giving others write permissions or else moving it to the georust org. The advantage of having a private repo is that I don't have to formulate my criterias for adding a link to the list and just let my experience/intuition decide.

Other awesome lists have more categories for documentation, blogs, tutorials, etc. but I think this is a major gap in our ecosystem.
